src/service_name.json

```diff
@@ -428,10 +433,5 @@
     "Command": "az dataprotection",
     "AzureServiceName": "Azure Data Protection",
     "URL": ""
-  },
```

Comment:
Hmm are these changes expected? Could you re-base you branch to upstream to ensure we don't make any changes here?